CHANGELOG.md

```diff
@@ -0,0 +1,38 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+## 0.1.0 – 2017-08-XX
+### Breaking changes
+- You need add two config parameter to `config.json`:
+  - `listen` defines the ip address and port Matterpoll will listen for requests. Default is `localhost:8505`
+  - `token` is the mattermost token. You should copy it from Integration > Slash Commands at your Mattermost server.
+
+### Added
+- Polls are posted as a bot user
+- You can use single and double quotation instead of backticks
+- We now verify the Mattermost token to make sure requests are valid
+- The ip address Matterpoll should listen for request can now we configured via `config.json`
+- You can choose the config file via `-c` option
+- Go 1.6 to 1.9 are now supported
+
+### Changed
+- Removed `-p` option. Choosing the port is now done via `config.json`
```

Comment:
I think that removing `-p` option is breaking change.

Reply:
:+1: